src/tracer.ts

```diff
@@ -0,0 +1,30 @@
+import tracer from "dd-trace"
+
+export function init() {
+  const ddServicePrefix = "positron"
+
+  tracer.init({
+    // Setting the service name to  `.undetected` so that we detect services that
+    // are running as part of the application, but aren't explicitly configured
+    // with `tracer.use`. We want to to explicitly configure services so that we
+    // can enforce our `.`-deliminited Service naming convention.
+    service: `${ddServicePrefix}.undetected`,
+    hostname: process.env.DATADOG_AGENT_HOSTNAME,
```

Comment:
A note about rolling this out...  Since the code will go out via CI running `hokusai deploy {Git-SHA}` and the config change adding the env var `DATADOG_AGENT_HOSTNAME` needs to be manually applied via `hokusai [staging|production] update`, perhaps default here to `datadog` as trace agent service is available in-cluster by this DNS name (tho it's not guaranteed to resolve to the IP of the container agent running on the same machine as the application's container - hence the use of the [downward api](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#the-downward-api)

Reply:
Don't totally following you here. If I patched the staging and production environments before rolling out the changes (via `hokusai [staging|production] update`), then the ENV vars would exist on the containers that will be created by the CI deploy, correct? Should that be the order of operations?

Reply:
@izakp and I synced IRL. We discussed the fact that a `hokusai staging update` depends on the local check-out of the Hokusai-relevant files. We concurred that setting the env first is a good approach though.

Reply:
Ah yes - that would be even simpler if you set up the env var before the deploy rolls out.  Just be sure to run `hokusai [staging|production| update` while checking out this commit.  This loose coupling still manages to be a pain point in the process.  Want to work on it in https://github.com/artsy/hokusai/issues/61